algorithms/hhl/hhl.ipynb

```diff
@@ -0,0 +1,1367 @@
+{
```

Comment:
<p>As a final comment you can write that while the exact result has smaller depth, it is not useful for larger use-cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>